app/models/AlfJourneyConfig.scala

```diff
@@ -52,12 +52,18 @@ case class TimeoutConfig(timeoutAmount: Int,
                         )
 
 case class ManualAddressEntryConfig(
-                                     mandatoryAddressFields: MandatoryAddressFields
+                                     line1MaxLength: Int,
+                                     line2MaxLength: Int,
+                                     line3MaxLength: Int,
+                                     townMaxLength: Int,
```

Comment:
these are technically optional, but since we have a known limit it does make sense to specify that here 👍